openshift/template.yaml

```diff
@@ -27,7 +27,17 @@ objects:
             value: "${GUNICORN_CMD_ARGS}"
           image: "${DOCKER_REGISTRY}/${DOCKER_IMAGE}:${IMAGE_TAG}"
           name: f8a-3scale-admin-gateway
-          ports:
+          - name: THREE_SCALE_API_ACCESS_KEY
```

Comment:
Now I noticed that this should actually go to the `env` section of the template, starting on line 25: https://github.com/fabric8-analytics/3scale-admin-gateway/pull/9/files#diff-8c255b087ddc69d83b81d4fb3a18c9cbL25

I'll also need env. variable for `THREE_SCALE_API_HOST_URL`, which can be taken from the config map:
```
          - name: THREE_SCALE_API_HOST_URL
            valueFrom:
              configMapKeyRef:
                name: bayesian-config
                key: three-scale-api-host-url
```